### PR TITLE
Rosbot

### PR DESCRIPTION
**Pull request by radeknh**
Please add a new ROS-based robot to your robot page. Thanks!
I don't have the environment for local testing but I hope everything will work.

### CODE REVIEW

File: _posts/2017-07-31-rosbot.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: ROSbot
+date: 2017-07-31
+description:
```
Comment (tfoote) [inline _posts/2017-07-31-rosbot.md]: Please fill in the description with something short to go in the top right above the tags: 
image

File: _posts/2017-07-31-rosbot.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+icon: /assets/img/robots/rosbot/rosbot_icon.jpg
+image: /assets/img/robots/rosbot/rosbot.jpg
```
Comment (tfoote) [inline _posts/2017-07-31-rosbot.md]: This filename is wrong. See the screenshot in my other comment.

File: _posts/2017-07-31-rosbot.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+---
+
+ROSbot is an open source autonomous robot platform.
+
+ROSbot in action on the ROS workshops:
+
+<iframe src="https://player.vimeo.com/video/224989128?title=0&byline=0&portrait=0" width="640" height="360" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+
+ROSbot, along with an in-depth ROS tutorial that Husarion provides: 
+ * https://husarion.com/core2/tutorials/ros-tutorials/1-ros-introduction/
```
Comment (tfoote) [inline _posts/2017-07-31-rosbot.md]: These should be links in markdown use []() syntax.

**Comment by radeknh** [comment]
Thank you for review. Changes made, please check.

**Pull request merged**